.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: false
+language: ruby
+cache: bundler
+rvm:
+  - 2.3.0
+  - ruby-head
+  - 2.2
```

Comment:
I think `2.2` is not really useful here... let's maintain a stable (`2.3.0` atm) and a variable (`ruby-head`) version.